### PR TITLE
Add Liberty Prime

### PR DESCRIPTION
**Pull request by MichaelHHall**
This PR adds the functionality for Terminus to spin up a docker image of liberty prime and run it. It even includes a (untested) functionality to add a branch parameter to make live testing easier. 

As an aside, I think the next thing I'm gonna do is work on streamlining the docker build process by creating a base dependency image and only rebuilding that when there's been a dependency change (should be very infrequent) and then using that as the base for the actual liberty image.

### CODE REVIEW

File: Modules/LibertyModule.cs [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+            string lib_dir = Config["LibertyPrimeDir"];
+            System.Environment.SetEnvironmentVariable("LIBERTY_DIR",lib_dir);
+            System.Environment.SetEnvironmentVariable("LIBERTY_BRANCH", branch);
+
+            ProcessStartInfo psi = new ProcessStartInfo();
+            psi.FileName = "Automation/activate_liberty_prime.sh";
+            psi.UseShellExecute = false;
+            psi.RedirectStandardOutput = true;
+
+            Process p = Process.Start(psi);
```
Comment (JOBBIN9422) [inline Modules/LibertyModule.cs:35]: wrap processes in `using` statement (e.g. `using (Process p = Process.Start(psi)) { ... }`)

File: Modules/LibertyModule.cs [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        {
+            // In the future, we may have persistent data to be saved from the docker container
+            // If that happens, we should first copy that data out of the container and into
+            // the persistent_files dir. This could happen here or in the deactivate script.
+            ProcessStartInfo psi = new ProcessStartInfo();
+            psi.FileName = "Automation/deactivate_liberty_prime.sh";
+            psi.UseShellExecute = false;
+            psi.RedirectStandardOutput = true;
+
+            Process p = Process.Start(psi);
```
Comment (JOBBIN9422) [inline Modules/LibertyModule.cs:53]: ditto, wrap process in `using` statement

**Review by JOBBIN9422 — CHANGES_REQUESTED** [review]

**Comment by ct-bess** [comment]
Any update on this?

**Comment by ct-bess** [comment]
SHUCKS DANG THNX

**Pull request merged**